### PR TITLE
fix(LIVE-10131): swap tron swapping issue with other pairs than eth and edge cases to investigate part 2

### PR DESCRIPTION
**Pull request by cng-ledger**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Takes additional fee off the swappable amount when max toggle is on, to prevent insufficient balance error

### ❓ Context

- **JIRA or GitHub link**: [LIVE-10131](https://ledgerhq.atlassian.net/browse/LIVE-10131)

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-10131]: https://ledgerhq.atlassian.net/browse/LIVE-10131?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/ledger-live-common/src/exchange/swap/hooks/useUpdateMaxAmount.ts [hunk 1]
```diff
@@ -41,8 +41,17 @@ export const useUpdateMaxAmount = ({
...
+              additionalFees = new BigNumber(5000000000000000); // 0,005 ETH/BNB/MATIC
+              break;
+            case tx.family === "tron" && next:
+              // https://github.com/tronprotocol/documentation/blob/master/TRX/Tron-overview.md
+              // 1 TRX=1,000,000 sun.
+              additionalFees = new BigNumber(1).multipliedBy(1_000_000); // 1 TRX
+              break;
+            default:
+              break;
           }
```
[the remark this answers is not in the document]
Comment (kallen-ledger) [reply libs/ledger-live-common/src/exchange/swap/hooks/useUpdateMaxAmount.ts:55]: Think I agree with @andreicovaciu on this.
